### ISSUE TITLE
Travis won't work

### ISSUE THREAD

**Issue by AndreG-P** [ag-gipp/MathMLConverters#20]
Travis doesn't work and I don't know why. Last Travis run is from 10. January...
@vstange any ideas?

**Comment by vstange** [comment]
Have reset the hook. Travis actually did not get any more requests. Could possibly have had something to do with the account conversion of ag-gipp.

**Comment by vstange** [comment]
@AndreG-P made a new test branch and tried to correct the mathoid test cases:
https://github.com/ag-gipp/MathMLConverters/tree/test

Do you know or can you check what changed in LateXML and if we can just accept the result?

**Comment by vstange** [comment]
PR #22 takes care of the testcases

**Comment by AndreG-P** [comment]
Ahh, that makes sense. We improved LaTeXML for our gold standard (for example, make power unambiguous). We didn't know we destroy some tests with those changes.
Furthermore, the server is running on the latest set of DLMF macros. It looks like the current configuration with this new set do not automatically include the macros from the DRMF dataset (which includes the `\Int`). I ran some tests and just adding `--preload: DRMFfcns.sty` __after__ the import of the `DLMFfcns.sty` should fix this problem.
I will open another issue for this problem.
